src/impy/models/epos.py

```diff
@@ -78,6 +78,8 @@ def __init__(self, event_kinematics, seed=None, logfname=None):
             self._output_frame = "laboratory"
 
         k = event_kinematics
+
+        _check_model_data_files("epos")
         datdir = path.join(base_path, epos_conf["datdir"])
```

Comment:
`datdir = CachedPath("epos")`

Reply:
CachedPath combines two steps that always have to go together anyway for the code to be correct, that's why it is a good design.

Reply:
As we need also url and the path of the directory to unpack the interface should be at least

```python
datdir = CachedPath(path.join(base_path, epos_conf["datdir"]), "epos")
```
If we use the parameter for model ("epos" in this case), we cannot use CachedPath for path construction as:

```python
datdir = CachedPath(base_path, "epos")/ CachedPath(epos_conf["datdir"]), "epos")
```
Then why we need CachedPath? Maybe it is just enough to have a function with interface `((path.join(base_path, epos_conf["datdir"]), "epos")`?

Reply:
The CachedPath can look up the base_path automatically, we don't need to do this at the call site. Ideally, this should just be `CachedPath("epos")` or see my comment above regarding the function `_cached_path`